api-reference/v2/api-reference-template.md

```diff
@@ -0,0 +1,188 @@
+The {product name} APIs provide {x} versions of API endpoints...
+
+## Pagination
+
+{This section is optional.}
+
+You can use {xx} to paginate the results returned by the {product name} APIs. If not specified, the default values are ...
+
+## Rate limiting and throttling
+
+{This section is optional.}
+
+The {product name} APIs use a {xx} rate limiting strategy. The maximum number of requests allowed to access a {resource/endpoint/..} is {xx} requests per {time period}.
+
+If the number of requests exceeds that limit, an HTTP `429 Too Many Request` error is returned.
+
+## Status code
+
+The {product name} APIs use standard HTTP response codes.
+
+| Status code | Message           | Description   |
+|-------------|-------------------|---------------|
+| 200 `OK`    | Request succeeds. | {description} |
+|             |                   |               |
+|             |                   |               |
+
+# {Resource name}
+
+The {resource name} is used to {xxx}.
+
+## Data model
+
+| Attribute | Type   | Required/Optional | Description                |
+|-----------|--------|-------------------|----------------------------|
+| {id}      | string | REQUIRED          | {Unique identifier of xx.} |
+| {name}    | string | Optional          | {Name of xx}.              |
+|           |        |                   |                            |
+
+
+## Example
+
+```
+<EXAMPLE>
+```
+
+## Endpoints
+
+Use the following endpoints to interact with the {resource name} entities.
+
+| Method | Endpoint name                           | Description           |
+|--------|-----------------------------------------|-----------------------|
+| POST   | {[Endpoint name A](link_to_endpoint_a)} | Creates a {resource}. |
+| GET    | {[Endpoint name B](link_to_endpoint_b)} | Returns a {resource}. |
+|        |                                         |                       |
+
+
+# {Endpoint name}
+
+{Provide a one-line description of what the API does. Starts with a verb.}
+
+## Endpoint
+
+```bash
+{METHOD} /{request_url}/{{path_parameter}}
+```
+
+## Description
+
+This endpoint ...
+
+{This paragraph is optional.} This endpoint has been deprecated. Use {the recommended endpoint} instead. For more information about how to migrate to {the recommended endpoint}, see [{the migration guide}](link).
+
+{This paragraph is optional.} The maximum number of calls to this API endpoint is {xx} per minute. For more information about API rate limiting/throttling, see [{the topic}](example).
+
+
+## Authorization
+
+The [{authorization method}](#authorization) is required for each API request.
+
+(Optional) Calling this endpoint also requires the {xx} permission.
+
+
+## Request schema
+
+### Path parameters
+
+{This section is optional.}
+
+| Path parameter | Type   | Required/Optional | Description                |
+|----------------|--------|-------------------|----------------------------|
+| {id}           | string | REQUIRED          | {Unique identifier of xx.} |
```

Comment:
```suggestion
| {id}           | string | Required          | {Unique identifier of xx.} |
```
* Suggest that we be consistent in our use of case, and I err on using lower case rather than upper case, as I don't think there is value gained by emphasizing with the capitals.